tests/integration/cattletest/core/test_container.py

```diff
@@ -629,6 +629,32 @@ def test_container_exec(admin_client, sim_context):
     admin_client.delete(c)
 
 
+def test_container_logs(admin_client, sim_context):
+    c = admin_client.create_container(imageUuid=sim_context['imageUuid'],
+                                      requestedHostId=sim_context['host'].id)
+    c = admin_client.wait_success(c)
+
+    assert callable(c.logs)
+
+    resp = c.logs(follow=True, lines=300)
+
+    assert resp.url is not None
+    assert resp.token is not None
+
+    jwt = _get_jwt(resp.token)
+
+    assert jwt['exec']['AttachStdin']
+    assert jwt['exec']['AttachStdout']
+    assert jwt['exec']['Tty']
+    assert jwt['exec']['Cmd'] == 'echo hello'
```

Comment:
I forgot to change that too sorry good catch Will fix it tomorrow i really need to be more cautious about clean up